File: python/7 kyu/355.py
# https://www.codewars.com/kata/557cffec8c3e8e55cc00010f/train/python

# Passed

def count_zeros(x):
    zeroes_count = 0
    for num in range(1, x + 1):
        if "0" in str(num):
            zeroes_count += str(num).count("0")
            
    return zeroes_count

output = count_zeros(200)
print(output)

File: nodejs/7 kyu/355.js
// https://www.codewars.com/kata/66cdc6ab9e7a9f009e0ca8f6/train/javascript

// Passed

function canSnailReachEnd(length, speed, lengthIncreases) {
  let distCoveredInAYear = speed * 365 * 24 * 60;
  let lengthInAYear = length + lengthIncreases * 365 * 24 * 60;
  let result = distCoveredInAYear > lengthInAYear;
  return result;
}

const output = canSnailReachEnd(10, 2, 1);
console.log(output);